api/resources/translations/messages-en.properties

```
@@ -391,6 +391,8 @@ branding.title.field = Title
 branding.title.field.help = This text will be displayed in the browser tab and as the PWA title.
 branding.icon.field = Large icon
 branding.icon.field.help = Will show for PWA installations. Should be at least 144 pixels square.
+browser.compatibility.title = Your Browser is outdated.
+browser.compatibility.description = Your device needs to be updated before it will work with CHT.
```

```suggestion
browser.compatibility.title = Your Browser is outdated
browser.compatibility.description = Update your browser to use the application.
```

Many CHWs don't know that it's called "CHT" so we usually use "app" or "application" 
Translation in Spanish:
```
browser.compatibility.title = Su navegador está desactualizado
browser.compatibility.description = Actualice su navegador para utilizar la applicación
```
You can find other teammates in Product or Medic in general, to give you the translation of these